### PR TITLE
Task 12.7

### PR DESCRIPTION
**Pull request by AdamPawlinski**
Task 12.7 finished. Please accept merging.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <head>
```
Comment (matwjablonski) [inline index.html:3]: lang for html

File: js/App.js [hunk 2]
```diff
@@ -1,27 +1,23 @@
...
-	return str;
-}
-
-// TWORZENIE NOWYCH EGZEMPLARZY KOLUMN
-var todoColumn = new Column('Do zrobienia');
-var doingColumn = new Column('W trakcie');
-var doneColumn = new Column('Skończone');
+var baseUrl = 'https://kodilla.com/pl/bootcamp-api';
+var myHeaders = {
+  'X-Client-Id': '3026',
```
Comment (matwjablonski) [inline js/App.js:3]: id can be a number ;)

File: js/Card.js [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+	this.name = name || 'No name given';
 	this.element = createCard();
 
 	function createCard() {
 		var card = $('<li class="card"></li>');
 		var cardDeleteBtn = $('<button class="btn-delete">x</button>');
 		var cardDescription = $('<p class="card-description"></p>');
-		
+
 		cardDeleteBtn.click(function(){
```
Comment (matwjablonski) [inline js/Card.js]: better use method `on` instead of `click`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
finished. ask for acceptance.

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**